### PR TITLE
Fix grammar issues with README.md

### PR DESCRIPTION
**Pull request by tabokie**
### What changes were proposed in this pull request?

Fix grammar issues with README.md

### Why are the changes needed?

Some sentences don't sound natural to me. I try my best to improve them, feel free to offer advice.

### Will break the compatibility? How if so?

No.

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

No code.

### Checklist

- [ ] I've run the tests to see all new and existing tests pass.
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above.
- [ ] I've informed the technical writer about the documentation change if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-- Highly compatible with MySQL
-    OceanBase Database is compatible with MySQL protocol and syntax. You can access to OceanBase Database by using MySQL client.
+    OceanBase Database scales transparently to applications and balances the system load automatically. It can manage more than 1500 nodes in a cluster. The data volume can reach petabytes. The records in a single table can be more than a trillion rows.
+- High compatibility with MySQL
+    OceanBase Database is compatible with MySQL protocol and syntax. You can access OceanBase Database by MySQL client.
 - High performance
     OceanBase Database supports quasi memory level data change and exclusive encoding compression. Together with the linear scalability, OceanBase Database provides high performance.
 - Low cost
-    OceanBase Database uses PC servers and cheap SSDs. Its high storage compression ratio and high performance also reduce the storage cost and the computing cost.
+    OceanBase Database can leverage PC servers and cheap SSDs. Its high storage compression ratio and high performance also reduce the storage and computing costs.
```
Comment (Amber1990Zhang) [inline README.md:16]: Thanks for your contribution. How about
OceanBase Database leverages PC servers and cheap SSDs.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 - Multi-tenancy
     OceanBase Database supports native multi-tenancy architecture. One cluster supports multiple businesses. Data is isolated among tenants. This reduces the deployment, operation, and maintenance costs.
 
-OceanBase Database supports the entire core business of Alipay and the core systems of hundreds of financial institutions, such as banks and insurance companies.
+OceanBase Database is supporting the entire core business of Alipay and the core systems of hundreds of financial institutions, such as banks and insurance companies.
```
Comment (Amber1990Zhang) [inline README.md:20]: I think the original is ok.

File: README.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ Contributions are warmly welcomed and greatly appreciated. Here are a few ways y
 
 ## Support
 
-In case you have any problems when using OceanBase Database, welcome reach out for help:
+In case you have any problems when using OceanBase Database, welcome to reach out for help:
```
Comment (Amber1990Zhang) [inline README.md:65]: nice catch

**Review by Amber1990Zhang — COMMENTED** [review]
Thanks for your contribution. Some minors.

**Comment by tabokie** [comment]
Comments addressed, and I added a few line breaks since #75 is closed.

File: README.md [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
 OceanBase Database has these features:
 
-- High availability
-    Single server failure recovers automatically. OceanBase Database supports cross-city disaster tolerance for multiple IDCs and zero data loss. OceanBase Database meets the financial industry Level 6 disaster recovery standard (RPO=0, RTO<=30 seconds).
-- Linear scalability
-    OceanBase Database scales transparently to applications and balances the system load automatically. Its cluster can contain more than 1500 nodes. The data volume can reach petabytes. The records in a single table can be more than a trillion rows.
-- Highly compatible with MySQL
-    OceanBase Database is compatible with MySQL protocol and syntax. You can access to OceanBase Database by using MySQL client.
-- High performance
+- High availability</br>
```
Comment (Amber1990Zhang) [inline README.md:7]: no html in markdown files

**Comment by LINxiansheng** [comment]
Thanks for your contribution.
The readme change too frequently and has too many conflicts.

**Comment by tabokie** [comment]
I don't see any conflict at all. Just close it if you want, but stop making excuses like this.

**Pull request closed without merging**